internal/backends/testing/lib.rs

```diff
@@ -279,6 +279,27 @@ mod for_unit_test {
         )
     }
 
+    pub fn send_keyboard_char_repeat<
```

Comment:
i'm not sure if it is worth adding this function.  The test can call  `instance.window().dispatch_event()`  directly.
Sending repeated event doesn't seem something like we need a convenience in the tests.
(Most of these function were added before dispatch_event was part of the public API)

Reply:
Ah yes that's better, done.